text_formatter.go

```diff
@@ -217,9 +221,12 @@ func (f *TextFormatter) printColored(b *bytes.Buffer, entry *Entry, keys []strin
 	}
 
 	levelText := strings.ToUpper(entry.Level.String())
-	if !f.DisableLevelTruncation {
+	if !f.DisableLevelTruncation && !f.PadLevelText {
 		levelText = levelText[0:4]
 	}
+	if f.PadLevelText {
+		levelText = fmt.Sprintf("%-7s", levelText)
```

Comment:
I'd rather that to not be hard coded, so that when/if change the available trace levels this should not need to be updated.
This could be done in an init function by iterating on AllLevels to find the largest one.

Reply:
I had that same idea while writing this! I'll add that 👍

Reply:
@dgsb I took a short break from this to do other stuff, but now I have returned!

I added https://github.com/sirupsen/logrus/pull/977/commits/693469de8f5d1e17ff6f3d8b479f36e5f97c5b7d and https://github.com/sirupsen/logrus/pull/977/commits/8ba442aca65c8333a2c78c566e064cc0e8ad1f88 since ya last looked at this PR 👀. Those commits add the dynamic max length thing!